packages/calcite-components/src/components/dropdown/dropdown.e2e.ts

```diff
@@ -1060,7 +1060,7 @@ describe("calcite-dropdown", () => {
       expect(item7).not.toHaveAttribute("aria-checked");
     });
 
-    it("item selection should work when placed inside shadow DOM (#992)", async () => {
+    it.skip("item selection should work when placed inside shadow DOM (#992)", async () => {
```

Comment:
Currently debugging this test with @jcfranco. However, I was able to "un-skip" two other tests with the code changes I made.

Reply:
For posterity, the issue was the dropdown not being fully initialized by the time the item was clicked, so it was causing some event handlers to fire that assumed the dropdown being ready.